exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/FragmentsRunner.java

```diff
@@ -61,7 +61,7 @@
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FragmentsRunner.class);
   private static final ControlsInjector injector = ControlsInjectorFactory.getInjector(FragmentsRunner.class);
 
-  private static final long RPC_WAIT_IN_MSECS_PER_FRAGMENT = 5000;
+  private static final long RPC_WAIT_IN_MSECS_PER_FRAGMENT = 30000;
```

Comment:
These changes were added as a separate commit to fix issues with our functional tests and build so that I could test this change. I've opened separate PRs for these changes so I'll remove them from this PR.